listenbrainz/webserver/__init__.py

```diff
@@ -94,8 +95,17 @@ def create_app(debug=None):
     # Database connections
     from listenbrainz import db
     from listenbrainz.db import timescale as ts
-    db.init_db_connection(app.config['SQLALCHEMY_DATABASE_URI'])
-    ts.init_db_connection(app.config['SQLALCHEMY_TIMESCALE_URI'])
+
+    # If we're running tests, overwrite the given DB configuration from the config and disregard the
+    # configuration, since that configuration could possibly point a different (production) DB.
+    if app.config['TESTING']:
```

Comment:
some tests set TESTING=False to test for certain conditions that are disabled when TESTING is true.

Reply:
need to think how to reconcile the two.

Reply:
FWIW, I've tested with invalid connection strings in my config.py and the tests ran just fine, so there was no attempting to connect to a resource defined by the strings.

Reply:
Yes, those tests currently don't access the DB.